src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js

```diff
@@ -29,12 +34,32 @@ class ImageContainer extends Component {
 		}
 	}
 
+	resizeImage(width, height) {
+		if((width/height) > 1){
+			return {"width": "auto", "height": 90, "top": 0, "left": parseInt((-1*(((width/height)*90)-90)/2), 10)};
```

Comment:
Can you add a comment here explaining what the math is? Also, I would recommend storing `90` in a local variable called `dimension`/`size` if you need it more than once (and maybe even if you need it only once).

Reply:
Yeah! will do!